# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/wine/hotfix-gcc14.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- wine-9.13/dlls/msvcrt/except_arm.c.vanilla	2024-07-25 20:23:24.230919742 +0200
+++ wine-9.13/dlls/msvcrt/except_arm.c	2024-07-25 20:23:39.559919557 +0200
@@ -66,7 +66,7 @@
 /*******************************************************************
  *		call_unwind_handler
  */
-void *call_unwind_handler( void *handler, ULONG_PTR frame, DISPATCHER_CONTEXT *dispatch )
+void *call_unwind_handler( void *handler, uintptr_t frame, DISPATCHER_CONTEXT *dispatch )
 {
     TRACE( "calling %p frame %Ix\n", handler, frame );
     return call_exc_handler( handler, frame, 0x100, dispatch->NonVolatileRegisters );
